nixos/doc/manual/installation/installing.xml

```diff
@@ -469,15 +470,10 @@ Retype new UNIX password: ***</screen>
      a previous configuration if something goes wrong.
     </para>
     <para>
-     You should log in and change the <literal>root</literal> password with
-     <command>passwd</command>.
-    </para>
-    <para>
-     You’ll probably want to create some user accounts as well, which can be
-     done with <command>useradd</command>:
-<screen>
-<prompt>$ </prompt>useradd -c 'Eelco Dolstra' -m eelco
-<prompt>$ </prompt>passwd eelco</screen>
+     Use your declared user account to log in. If you didn't declared one,
+     you should still be able to log in using <literal>root</literal> user.
```

Comment:
Maybe add a sentence like “Note that some graphical display managers like SDDM do not allow root login by default so you might need to switch to TTY.”

That would check all the points on the issue (except for phasing out root login but that probably deserves an issue of its own).

Reply:
Done! Thanks for the suggestion!